### DISCUSSION TITLE
Welcome

### DISCUSSION THREAD

**Discussion by john30** [john30/ebusd#517] [Announcements]
This is supposed to be the new area for discussions around ebusd to avoid spreading to too many different places.
Feel free to ask questions, contribute to Q&A, tell a story of what was accomplished and how.

**Comment by FrigerioSergio** [comment]
Hello John30. I'm using Home Assistant and would like to control my heating system. I have already "preordered" one of your EBUS Adapter 3 but it's seems that they are not available anymore.

Will the Esera Ebus coupler (https://www.esera.de/shop/en/products/ebus/135/ebus-coupler-usb) works with your project?

Thanks and best regards,

Sergio

**Reply by john30** [reply]
see here: https://github.com/john30/ebusd/wiki/6.-Hardware
please don't misuse this topic for unrelated questions. instead open a new discussion

**Comment by rgilbertw** [comment]
Question: How can I configure ebusd to make sure that data are only read, and the only data sent to the devices are the requests for the scans.
In combination ebusd with iobroker I got the impression, that data are unwillingly sent to the devices what caused problems.

**Comment by chrisbev** [comment]
UPDATE: new poster here, this now looks not the place to post will try the q&a section so please ignore. 
{{Hi I'm trying to write SetModeOverride to vaillant 625 using Home Assistant, I have the Ebusd HA addon I am told I need bai.308523.inc but can't find the correct config file with SetModeOveride in looked in john30/ebusd-configuration/ebusd-2.1.x/vaillant/.  [ installed addon with LukasGrebe link and doesn't have that line].}}
